appengine/analytics/app.js

```diff
@@ -48,7 +48,7 @@ function trackEvent (category, action, label, value, cb) {
   };
 
   return got.post('http://www.google-analytics.com/collect', {
-    form: data
+   ...data
```

Comment:
This operator only works in Node v8.6+, but we strive to make sure our samples work with the oldest LTS version of Node, which until the end of April is Node v4. Can you update your PR to be Node v4 compatible?

Reply:
Hi @jmdobry,

Makes sense, updated to use Object.assign instead.